### PR TITLE
PM-1577 - Update transaction generator values.yaml

### PR DESCRIPTION
**Pull request by kaozenn**

### CODE REVIEW

**Review by simisimis — APPROVED** [review]
this looks proper!

**Comment by Smorci** [comment]
I know this is a draft, but I did try it out and have an observation.

**issue**: The pre-commit hook is not generating the README for `mina-transactions-generator`.

The `helm-docs` binary mentions this about the flag `--chart-to-generate` 
```
--chart-to-generate strings                          List of charts that will have documentation generated. Comma separated, no space. Empty list - generate for all charts in chart-search-root
```
I did this diff and ran `pre-commit run` after that and it generated the README.md
```
diff --git a/.pre-commit-config.yaml b/.pre-commit-config.yaml
index 73efce5..21e8c47 100644
--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -7,7 +7,7 @@ repos:
       - id: helm-docs-built
         args:
           # Comma separated list, no space
-          - --chart-to-generate=gpt-survey-summarizer, mina-transactions-generator
+          - --chart-to-generate=gpt-survey-summarizer,mina-transactions-generator
 
           # The `./` makes it relative to the chart-search-root
           - --template-files=./README.md.gotmpl
diff --git a/mina-transactions-generator/values.yaml b/mina-transactions-generator/values.yaml
index 3167ada..abbecaf 100644
--- a/mina-transactions-generator/values.yaml
+++ b/mina-transactions-generator/values.yaml
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-# -- The number of replicas
+# -- The number of replicas test
 replicaCount: 1
 
 image:
 ```

**Comment by kaozenn** [comment]
Thanks @Smorci That's the way to go :)

File: mina-transactions-generator/values.yaml [hunk 1]
```diff
@@ -2,44 +2,67 @@
...
 replicaCount: 1
 
 image:
+  # -- The repository of the image
   repository: 673156464838.dkr.ecr.us-west-2.amazonaws.com/mina-transactions-generator
+  # -- The pullPolicy used when pulling the image
   pullPolicy: IfNotPresent
-  # Overrides the image tag whose default is the chart appVersion.
-  tag: 0.1.2-5b82cae
+  # -- The tag of the iamge. Overrides the image tag whose default is the chart appVersion.
```
Comment (simisimis) [inline mina-transactions-generator/values.yaml:13]: suggestion: we could replace `iamge` with `image` 😆

Comment (kaozenn) [reply mina-transactions-generator/values.yaml]: Thnaks fro sopting tihs out! Rsevoled!

**Pull request merged**